File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{

    int quater = 0;
    int dime = 0;
    int nickle = 0;
    int penny = 0;
    int count = 0;
    float owed;

    // asking the user for how much owed.
    do
    {
        owed = get_float("how much are you owed \n");
    }
    while(owed < 0);


    // changing cent to dollar.
    int cents = round(owed * 100);
    printf("Change in Cent: %i\n", cents);


    while (cents >= 25)
    {
        quater++;
        cents = cents - 25;
    }
    while (cents >= 10)
    {
        dime++;
        cents = cents - 10;
    }
    while (cents >= 5)
    {
        nickle++;
        cents = cents - 5;
    }
    while (cents >= 1)
    {
        penny++;
        cents = cents - 1;
    }
    //summing up the total coins
    count = quater + dime + nickle + penny;
    printf("Total amount of coins: %i\n", count);
}

File: pset6/cash.py
from cs50 import get_float


owed = -1
quarter = dime = nickle = penny = 0

while owed < 0:
    owed = get_float("Change owed: ") * 100

quarter = int(owed / 25) if owed // 25 >= 1 else 0
owed -= quarter * 25

dime = int(owed/10) if owed // 10 >= 1 else 0
owed -= dime * 10

nickle= int(owed / 5) if owed // 5 >= 1 else 0
owed -= nickle * 5

penny = owed if owed // 1 >= 1 else 0
owed -= penny * 1

print(int(quarter + dime + nickle + penny))